content/blog/2020-02-glue-strings-and-tidy-eval/index.Rmarkdown

```diff
@@ -0,0 +1,119 @@
+---
+title: Tidy eval now supports glue strings
+author: Lionel Henry
+date: '2020-02-11'
+slug: glue-strings-and-tidy-eval
+categories:
+  - package
+tags: [package, r-lib, tidyverse]
+photo:
+  url: https://unsplash.com/photos/pcoKkN3NsX0
+  author: Modestas Urbonas
+---
+
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>"
+)
+iris <- tibble::as_tibble(iris)
+```
+
+[rlang 0.4.0](https://www.tidyverse.org/blog/2019/06/rlang-0-4-0/) introduced the curly-curly `{{` operator to simplify writing functions around tidyverse pipelines. The minor update 0.4.3 of rlang makes it possible to use `{` and `{{` to create result names in tidyverse verbs taking pairs of names and expressions.
+
+Install the latest version of rlang to make the new feature globally available throughout the tidyverse:
+
+```{r, eval = FALSE}
+install.packages("rlang")
+```
+
+
+## Tunnelling data-variables with curly-curly
+
+With the `{{` operator you can __tunnel__ data-variables (i.e. columns from the data frames) through env-variables (programming variables from the environment, in particular function arguments):
+
+```{r}
+suppressMessages(
```

Comment:
And, conversely, I'd be inclined to hide the mechanics of suppressing the tidyverse messaging, i.e. by setting `message = FALSE` for this chunk or setting `tidyverse.quiet`.